public/API/API_GetLeaderboardSubmissions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ *  API_GetLeaderboardSubmissions - returns submissions for a Leaderboard for the given ID
```

Comment:
nit: The more I think about it, the more I think we should rename this to API_GetLeaderboardEntries to match the nomenclature of the database and ORM model.

Reply:
Agreed, as I have looked at it it's felt weirder and weirder